chia/cmds/passphrase_funcs.py

```diff
@@ -118,7 +119,7 @@ def prompt_for_new_passphrase() -> Tuple[str, bool]:
 
 
 def read_passphrase_from_file(passphrase_file: TextIOWrapper) -> str:
-    passphrase = passphrase_file.read()
+    passphrase = passphrase_file.read().rstrip(os.environ.get("CHIA_PASSPHRASE_STRIP_TRAILING_CHARS", "\r\n"))
```

Comment:
What led you to make this configurable?

:)

Reply:
Ahh... existing users. I am not used to seeing such a complete PR description. Thanks!